tools/topology/topology2/include/pipelines/cavs/dai-copier-be.conf

```diff
@@ -0,0 +1,104 @@
+#
+# BE DAI copier pipeline
+#
+# All attributes defined herein are namespaced by alsatplg to
+# "Object.Pipeline.dai-copier-be.N.attribute_name"
+#
+# Usage: dai-copier-be pipeline object can be instantiated as:
+#
+# Object.Pipeline.dai-copier-be."N" {
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/pipeline.conf>
+
+Class.Pipeline."dai-copier-be" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		unique	"instance"
+	}
+
+	Object.Widget {
+		copier."1" {
+			type		dai_in
+			node_type	$HDA_LINK_OUTPUT_CLASS
+			num_audio_formats	3
+			# 32-bit 48KHz 2ch
+			Object.Base.audio_format.1 {
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size		"$[$obs * 2]"
+			}
+			# 32-bit 48KHz 4ch
+			Object.Base.audio_format.2 {
+				in_channels		4
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_channels		4
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$obs * 2]"
+				in_ch_cfg	$CHANNEL_CONFIG_3_POINT_1
+				in_ch_map	$CHANNEL_MAP_3_POINT_1
+				out_ch_cfg	$CHANNEL_CONFIG_3_POINT_1
+				out_ch_map	$CHANNEL_MAP_3_POINT_1
+			}
+			# 24-bit 48KHz 2ch
+			Object.Base.audio_format.3 {
+				in_bit_depth		32
+				in_valid_bit_depth	24
+				out_bit_depth		32
+				out_valid_bit_depth	24
+				dma_buffer_size		"$[$obs * 2]"
+			}
+		}
+
+		copier."2" {
+			type	buffer
+			copier_type	"module"
+			period_sink_count	2
+			period_source_count	0
```

Comment:
are these period counts really useful?

Reply:
I must admit I do not know.

Reply:
lets remove it. im pretty sure these are not used in ipc4